### PR TITLE
ansible: add libvirt role

### PR DESCRIPTION
**Pull request by FFY00**
Signed-off-by: Filipe Laíns <lains@archlinux.org>

### CODE REVIEW

**Comment by FFY00** [comment]
We only support login via ssh: `qemu+ssh://host/system`

File: roles/libvirt/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# https://github.com/ansible/ansible/issues/11024
+- name: remove all users from libvirt group
+  command: groupmems -g libvirt --purge
+
+- name: add admin users to libvirt
+  user:
+    name: '{{ item.key }}'
+    append: yes
+    groups: wheel
```
Comment (ljmf00) [inline roles/libvirt/tasks/main.yml]: shouldn't be `libvirt` group here?

Comment (FFY00) [reply roles/libvirt/tasks/main.yml]: Yes, good catch.

**Comment by ljmf00** [comment]
I've already tested this role on my local setup, it should be good to go.

**Review by ljmf00 — APPROVED** [review]

**Pull request merged**